src/current/v25.1/recommended-production-settings.md

```diff
@@ -22,6 +22,15 @@ For optimal cluster performance, Cockroach Labs recommends that all nodes use th
 
 We recommend running a [glibc](https://www.gnu.org/software/libc/)-based Linux distribution and Linux kernel version from the last 5 years, such as [Ubuntu](https://ubuntu.com/), [Red Hat Enterprise Linux (RHEL)](https://www.redhat.com/technologies/linux-platforms/enterprise-linux), [CentOS](https://www.centos.org/), or [Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs).
 
+New deployments should configure [transparent huge pages (THP)](https://www.kernel.org/doc/html/latest/admin-guide/mm/transhuge.html) with the `madvise` option:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+echo madvise > /sys/kernel/mm/transparent_hugepage/enabled
```

Comment:
These instructions don't survive restart, and I don't think they apply to existing processes. We used the instructions in this post for RedHat: https://access.redhat.com/solutions/46111

Maybe just say more general - permanently chaning the THP setting depends on the operating system. For Red Hat Enterprise Linux, see https://access.redhat.com/solutions/46111